cadquery/occ_impl/shapes.py

```diff
@@ -4998,6 +5082,19 @@ def _pts_to_harray(pts: Sequence[VectorLike]) -> TColgp_HArray1OfPnt:
     return rv
 
 
+def _pts_to_harray2d(pts: Sequence[Tuple[Real, Real]]) -> TColgp_HArray1OfPnt2d:
+    """
+    Convert a sequence of 2d points to a TColgp harray (OCCT specific).
```

Comment:
minor suggestion: change 2d to 2D.  I remember there was a PR some time ago that changed all docstring references to upper case for consistency.